electron/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "unified-auctions",
```

Comment:
```suggestion
  "name": "unified-auctions-app",
```

Reply:
I wasn't aware that the app will be named after this field, so I changed it back